lib/transmuxer.js

```diff
@@ -114,6 +114,8 @@ TransportParseStream = function() {
   TransportParseStream.prototype.init.call(this);
   self = this;
 
+  this.packetsWaitingForPMT = [];
```

Comment:
I've only been capitalizing the first letter of acronyms when camel-casing them elsewhere (e.g. `startPts`).